README.md

```diff
@@ -1,7 +1,7 @@
 # UUID
 
 [![License](https://img.shields.io/github/license/gofrs/uuid.svg)](https://github.com/gofrs/uuid/blob/master/LICENSE)
-[![Build Status](https://travis-ci.org/gofrs/uuid.svg?branch=master)](https://travis-ci.org/gofrs/uuid)
+[![Go](https://github.com/gofrs/uuid/actions/workflows/go.yml/badge.svg)](https://github.com/gofrs/uuid/actions/workflows/go.yml)
```

Comment:
The link update is 👍 but I would probably leave the label set to "Build Status"

Reply:
@dylan-bourque This label depends on the name of the GitHub Actions, so if we want it to be `Build`, we'll need to change `name` field in `go.yml`.

https://github.com/gofrs/uuid/blob/master/.github/workflows/go.yml#L1

Reply:
The alt text for the image in the Markdown has to match the name of the workflow?  That doesn't make sense and it's not called out that way in the docs anywhere that I can find.

Reply:
The `alt text` one, that's ok, I mistook it for the label shown on the badge.

Reply:
I fixed it 👍🏼